bibliopixel/project/project.py

```diff
@@ -76,23 +82,42 @@ def resolve_aliases(name):
 
     if not (driver or drivers):
         raise ValueError(
-            'The project has neither a "driver" nor a "drivers" sections')
+            'The project has neither a "driver" nor a "drivers" section')
 
     extend_path(path)
     maker = data_maker.Maker(**(maker or {}))
     make_object = functools.partial(importer.make_object, maker=maker)
 
-    builder = MultiMapBuilder(make_object)
-    driver_objects = builder.make_drivers(driver, drivers)
+    driver_objects = make_drivers(driver, drivers, make_object)
+
+    gen_coord_map = layout.pop('gen_coord_map', None)
```

Comment:
we've had coord_map forever. Used to be coordMap... gen_coord_map generates a coord_map! How is that complicated?
Ok... this is NOT pixel positions for SimPixel. Just wanted to get that out of the way... not three-mapping anything. Not here.
coord_map is simple.
coord_map is the map of 1D indices onto a Strip, Matrix, Cube, or Circle space.
everything under the hood is represented as a 1D list of colors. So a Matrix coord_map maps the pixel at (x,y) to the index i in the 1D color list.
gen_coord_map takes the similar params to what Matrix() layout has taken (well, for gen_matrix_multi) and generates a single unified coordinate map (again, (x,y) to i) for 1 or more sub matrices and their individual coord_maps. This means that you can have two drivers driving 2 8x8 matrices and one of them is rotated 90 degress and flipped, and then you can represent that in gen_coord_map and it figures it all out and then builds the unified coord_map, which is then used in the set() methods for the layouts. The layouts dole out pixels to drivers just in order. 0-49 for the 50 pixel driver, 50-149 for the second driver with 100 pixels. gen_coord_map allows you to take the separate arrangement (serpentine, flip, rotation) for each, and merge them into one so that the right pixel values end up in the right place.
we've had coord_map forever and we cannot change it because it would break 3.0.5 compat so we cannot change gen_coord_map because it's the logical name given what it does and what it changes. Basically if you use gen_coord_map it creates a coord_map entry in the layout before it all gets built.
This is all required when you have large displays like the 12K... that one is simpler because it's layout isn't really all that weird. but my big display Colossus has a weird layout so this is all required.
I don't know how to better explain it. 
Point being..... just trust me :P We need this.